examples/contrastive-image-text/run_clip.py

```diff
@@ -99,6 +226,7 @@ class ModelArguments:
     freeze_text_model: bool = field(
         default=False, metadata={"help": "Whether to freeze the text model parameters or not."}
     )
+    mediapipe_dataloader: bool = field(default=False, metadata={"help": "Turn on MediaPipe HW dataloading"})
```

Comment:
This argument is not related to the model, let's move it to `DataTrainingArguments` right below.

Reply:
fixed.